File: java-sample-app/src/main/java/com/aminography/primeadapter/sample/MyApplication.java
package com.aminography.primeadapter.sample;

import android.app.Application;
import android.support.v7.app.AppCompatDelegate;

import com.aminography.primeadapter.PrimeAdapter;

/**
 * Created by aminography on 7/14/2018.
 */
public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
    }

}

File: sample-app/src/main/java/com/aminography/primeadapter/sample/MyApplication.kt
package com.aminography.primeadapter.sample

import android.app.Application
import android.support.v7.app.AppCompatDelegate
import com.aminography.primeadapter.PrimeAdapter

/**
 * Created by aminography on 7/14/2018.
 */
@Suppress("unused")
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)
    }

}